lib/cmps/sourcepoint-frame.ts

```diff
@@ -36,7 +36,9 @@ export default class SourcePoint extends AutoConsentCMPBase {
   }
 
   async detectPopup() {
-    return true;
+    // check for the paywall button, and bail if it exists to prevent broken opt out
+    await waitForElement(".sp_choice_type_11,.sp_choice_type_12,.sp_choice_type_13,.sp_choice_type_ACCEPT_ALL", 2000);
+    return !elementExists('.sp_choice_type_9');
```

Comment:
Will the opt-in mode still work?

Reply:
No. For sites using the subscription button, we just consider the popup closed, and both opt-in and opt-out won't happen. Not sure that there is another way to handle this without making changes to the API.

For the meantime, I think it's worth making this change, so we don't fail with opt-outs on these sites.